### ISSUE TITLE
Use your triplet loss to finetune arcface recognition model

### ISSUE THREAD

**Issue by azadehashouri** [alfonmedela/TripletSemiHardLoss-PyTorch#4]
Hello dear Alfonmedela.

As far as I remember, I had asked you question many years ago in another repository. :))

Now, I'm working on finetuning Arcface-torch recognition model. My custom dataset has 2 images per identity and the number of total identities is about 4000.

Do you think I can use the triplet loss to finetune it? And how?

Thanks in advance
